bullet_train-internationalization/sig/bullet_train/internationalization.rbs

```diff
@@ -0,0 +1,6 @@
+module BulletTrain
+  module Internationalization
+    VERSION: String
+    # See the writing guide of rbs: https://github.com/ruby/rbs#guides
+  end
+end
```

Comment:
Do we need this? (I don't know a lot about rbs, but it doesn't seem like it's adopted by the project.)